src/structures/StageInstance.js

```diff
@@ -67,6 +67,7 @@ class StageInstance extends Base {
       /**
        * Whether or not stage discovery is disabled
        * @type {?boolean}
+       * @deprecated See https://github.com/discord/discord-api-docs/pull/4296 for more information
```

Comment:
Could you `@link` this?

Reply:
That will render the link fine on the website but totally unclickable in IntelliSense. The correct way is `@see <link>` but I chose to follow #7107 because I don't know how the website will render `@see` given that the whole of discord.js does either `{@link <text|link>}` or `@see {@link <text|link>}` and both of them render the link unclickable in IntelliSense.

Reply:
Yeah most links use `{@link}` and I could've sworn the unclickable links issue was solved but I guess that was a different issue or tag